platform/src/main/java/org/stellar/anchor/platform/service/TransactionService.java

```diff
@@ -196,6 +196,7 @@ void updateSepTransaction(PlatformTransactionData patch, JdbcSepTransaction txn)
     txnUpdated = updateField(patch, txn, "externalTransactionId", txnUpdated);
     // update stellar_transactions
     txnUpdated = updateField(patch, txn, "stellarTransactions", txnUpdated);
+    txnUpdated = updateField(patch, txn, "stellarTransactionId", txnUpdated);
```

Comment:
The intention is to have the business send a `stellarTransactions` list containing the stellar transaction id. It doesn't make sense to have both a single transaction ID field and a list.

Reply:
Yes it can be removed